applications/rag/variables.tf

```diff
@@ -79,8 +81,8 @@ variable "rag_service_account" {
 }
 
 variable "create_gcs_bucket" {
-  type = bool
-  default = false
+  type        = bool
+  default     = false
```

Comment:
shouldnt it be default true? or jupyterhub/ray wont work

Reply:
default false is for marketplace..